packages/next/package.json

```diff
@@ -67,6 +67,7 @@
   "dependencies": {
     "@babel/runtime": "7.15.4",
     "@hapi/accept": "5.0.2",
+    "@napi-rs/blake-hash": "1.2.0",
```

Comment:
Looks like this is going to add an additional 1+ MB https://packagephobia.com/result?p=%40napi-rs%2Fblake-hash

Reply:
I checked and its only 670KB on macOS so I'm guessing this is suffering from the [same linux bug](https://twitter.com/styfle/status/1458115734496759816) we saw before where both musl and gnu are installed.